examples/java-matter-controller/java/src/com/matter/controller/commands/common/Result.java

```diff
@@ -0,0 +1,23 @@
+/*
+ *   Copyright (c) 2023 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+
+package com.matter.controller.commands.common;
+
+public interface Result {
```

Comment:
Please explain the purpose of this class with a doc comment. Why is this called "result"? It is basically a boolean. Why not `BooleanResult`? What does true/false in a result represent? Should this be some form of CommandResult returning an enum of CommandResult::SUCCESS / CommandResult::FAILURE?

Reply:
actually this interface does not offer much benefits at this moment, just removed this class, now the code should become more concise now. thanks